### ISSUE TITLE
Users page

### ISSUE THREAD

**Issue by hjone72** [datkinson13/Student_Management_System#11]
The users page is currently looking fantastic however, once we add 600 users the page is going to start taking a very long time to load and will be difficult to navigate.

I suggest we implement https://github.com/yajra/laravel-datatables. This will:

- Offload information processing to the server
- Reducing the page load times
- Add filter capabilities via a search box.

**Comment by datkinson13** [comment]
Yeah, I was thinking of using pagination or something, but that package looks handy, especially the search box.

**Comment by hjone72** [comment]
I've used it before, it worked pretty well then but definitely open to other suggestions.
